tldr.py

```diff
@@ -138,7 +136,7 @@ def get_page_for_platform(command, platform):
     return data.splitlines()
 
 
-def download_and_store_page_for_platform(command, platform):
+def download_and_store_page_for_platform(command, platform, remote):
```

Comment:
There are references in the code to this function. Please pass the remote variable there too.